lang/python/python3-host.mk

```diff
@@ -5,12 +5,13 @@
 # See /LICENSE for more information.
 #
 
-# Note: include this after `include $(TOPDIR)/rules.mk in your package Makefile
+# Note: include this file after `include $(TOPDIR)/rules.mk in your package Makefile
 #       if `python3-package.mk` is included, this will already be included
 
 # For PYTHON3_VERSION
 python3_mk_path:=$(dir $(lastword $(MAKEFILE_LIST)))
 include $(python3_mk_path)python3-version.mk
+
```

Comment:
oh ; line got added back here;

Reply:
Not sure why the line got removed and added back :sweat_smile: I've fixed the commit to not remove it in the first place.